.github/workflows/deploy-firebase-acc.yml

```diff
@@ -18,14 +18,16 @@ jobs:
       - name: Configure
         uses: w9jds/firebase-action@master
         with:
-          args: functions:config:set clothingloop.base_domain="$BASE_DOMAIN" clothingloop.region="$REGION" clothingloop.admin_emails="$ADMIN_EMAILS"
+          args: functions:config:set clothingloop.base_domain="$BASE_DOMAIN" clothingloop.region="$REGION" clothingloop.admin_emails="$ADMIN_EMAILS" contactmail.to="$CONTACT_MAIL_TO" contactmail.cc="$CONTACT_MAIL_CC"
         env:
           ADMIN_EMAILS: ${{ secrets.ADMIN_EMAILS }}
           BASE_DOMAIN: ${{ secrets.BASE_DOMAIN }}
           FIREBASE_TOKEN: ${{ secrets.FIREBASE_TOKEN }}
           PROJECT_ID: ${{ secrets.FIREBASE_PROJECT_ID }}
           PROJECT_PATH: ./firebase/functions
           REGION: ${{ secrets.REGION }}
+          CONTACT_MAIL_TO: ${{ secrets.CONTACT_MAIL_TO }}
+          CONTACT_MAIL_CC: ${{ secrets.CONTACT_MAIL_CC }}
```

Comment:
Consider making this a single setting. "to" can already be an array: https://medium.com/firebase-developers/firebase-extension-trigger-email-5802800bb9ea